### ISSUE TITLE
to throw_term with tuple argument fails

### ISSUE THREAD

**Issue by mspanc** [antonmi/espec#65]
I have the following spec

```
defmodule DBux.Transport.TCPSpec do
  use ESpec

  context ".connect/1" do
    context "if passed an empty list as options" do
      it "throws {:badopts, :hostname, :missing}" do
        expect(fn -> DBux.Transport.TCP.connect([]) end) |> to throw_term({:badopts, :hostname, :missing})
      end
    end
  end
end
```

It fails with the following error:

```
	1) DBux.Transport.TCPSpec DBux.Transport.TCPSpec .connect/1 if passed an empty list as options throws {:badopts, :hostname, :missing}
	/vagrant/spec/transport/tcp_spec.exs:6
	** (Protocol.UndefinedError) protocol String.Chars not implemented for {:badopts, :hostname, :missing}
    (elixir) lib/string/chars.ex:3: String.Chars.impl_for!/1
    (elixir) lib/string/chars.ex:17: String.Chars.to_string/1
    (espec) lib/espec/assertions/throw_term.ex:36: ESpec.Assertions.ThrowTerm.success_message/4
    (espec) lib/espec/example_runner.ex:33: ESpec.ExampleRunner.run_example/2
    (elixir) lib/enum.ex:977: anonymous fn/3 in Enum.map/2
    (elixir) lib/enum.ex:1261: Enum."-reduce/3-lists^foldl/2-0-"/3
    (elixir) lib/enum.ex:977: Enum.map/2
    (espec) lib/espec/runner.ex:69: ESpec.Runner.run_examples/1
```

I am using 0.8.6

**Comment by antonmi** [comment]
Fixed. I've published 0.8.7 with the fix.
Thank for using ESpec!

**Comment by mspanc** [comment]
Thank you for such quick fix!
